src/docs/01-Components/06-Feedback/01-notifications.md

```diff
@@ -52,6 +52,15 @@ secondaryKeywords: critical warning error success box
 {{render '@notification--error'}}
 ```
 
+## Notification with no shadow (info)
+<div class="element-preview">
+  <div class="element-preview__inner">{{render '@notification--with-no-shadow'}}</div>
+</div>
+
+```html
+{{render '@notification--with-no-shadow'}}
+```
+
 
 # HTML Guidelines
 

```

Comment:
Maybe one thing could be to add a little snippet in about "when to use it" with no shadow (embedded into cards?) UX wise - but that's the only thing I can think about

Reply:
I agreed. However, a lot of polishing will be done in the near future by the “new” design team and I expect a digital guideline will be formulate including use cases.